Geometry/HcalTowerAlgo/interface/HcalGeometry.h

```diff
@@ -104,6 +104,9 @@ class HcalGeometry : public CaloSubdetectorGeometry {
 			   CaloSubdetectorGeometry::IVec&   iVector,
 			   CaloSubdetectorGeometry::DimVec& dimVector,
 			   CaloSubdetectorGeometry::IVec& dinsVector ) const ;
+
+  const HcalTopology& topology() const { return theTopology; }
```

Comment:
@lgray - yep, I've asked the question before. IMHO, harmonising the other way would have been better, providing that the record dependencies are correct. Personally, I don't like it, but for now it's ok, I guess.